sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ## Current version
 ###  Feature Additions
```

Comment:
Change the version here too to 4.3.0. Make sure to leave the Current Version at the top of the file